sql/sql.y

```diff
@@ -351,7 +367,21 @@ func (s trainClause) JSON() string {
 "columns": %s,
 "save": "%s"
 }`
-	return fmt.Sprintf(fmter, s.estimator, s.attrs.JSON(), s.columns.JSON(), s.save)
+	ccs := []string{}
+	for _, cc := range s.ccs {
+		ccs = append(ccs, cc.JSON())
+	}
+	columns := "[\n" + strings.Join(ccs, ",\n") + "\n]"
+
+	return fmt.Sprintf(fmter, s.estimator, s.attrs.JSON(), columns, s.save)
+}
+
+func (s columnsClause) JSON() string {
```

Comment:
I vaguely remember that the `JSON` functions were for some old purpose are no longer needed. Could @tonyyang-svail please confirm.

Reply:
Right. It's safe to remove it. :)